cmd/runAsNonRoot.go

```diff
@@ -46,15 +47,65 @@ func checkRunAsNonRoot(container ContainerV1, result *Result) {
 	return
 }
 
+// Checks the PSC for RANR
+
+func checkRunAsNonRootPSC(podSpec PodSpecV1, result *Result) {
+	if reason := result.Labels["audit.kubernetes.io/allow-run-as-root"]; reason != "" {
+		if podSpec.SecurityContext == nil || podSpec.SecurityContext.RunAsNonRoot == nil || *podSpec.SecurityContext.RunAsNonRoot == false {
+			occ := Occurrence{
+				podHost:  podSpec.Hostname,
+				id:       ErrorRunAsNonRootFalseAllowed,
+				kind:     Warn,
+				message:  "Allowed setting RunAsNonRoot to false",
+				metadata: Metadata{"Reason": prettifyReason(reason)},
+			}
+			result.Occurrences = append(result.Occurrences, occ)
+		} else {
+			occ := Occurrence{
+				podHost:  podSpec.Hostname,
+				id:       ErrorMisconfiguredKubeauditAllow,
+				kind:     Warn,
+				message:  "Allowed setting RunAsNonRoot to false, but it is set to true",
```

Comment:
I think we want disjunct error messages, so that people know where the errors come from.